src/test/java/com/purbon/kafka/topology/clusterstate/YAMLStateProcessorTest.java

```diff
@@ -0,0 +1,38 @@
+package com.purbon.kafka.topology.clusterstate;
+
+import com.purbon.kafka.topology.roles.TopologyAclBinding;
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.kafka.common.resource.ResourceType;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class YAMLStateProcessorTest {
+
+  @Test
+  public void testWriteAndReadState() throws Exception {
+    final List<TopologyAclBinding> bindings =
+        Arrays.asList(
+            new TopologyAclBinding(ResourceType.TOPIC, "A1", "A2", "A3", "A4", "A5"),
```

Comment:
awesome! I love the tests here.

do you mind using more meaningful names instead of A1 or B1 ? thanks, this will make future understanding munch easier.